src/ProjectNeon/Assets/Data/Cards/Effects/Damage.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+/**
+ * Applies Damage to some Target, i.e., reduces the Target H.P. in the quantity value.
+ */
+public class Damage : Effect
+{
+    [SerializeField] private int quantity;
+    public override void Apply(Target target)
+    {
+        if (target.GetType() == typeof(Member))
```

Comment:
Never use Type interrogation or Casting. 

I thought we were going to use Composite pattern for this? Can't target have a List of Members? That will work for one or many. 

Then, we also don't need to cast to members.

Reply:
@SilasReinagel Sorry about that! I just made a quick dirty code. Now I 've corrected that.